client/reader/stats.js

```diff
@@ -98,11 +98,6 @@ export function recordTrack( eventName, eventProperties ) {
 		eventProperties = Object.assign( { subscription_count: subCount }, eventProperties );
 	}
 
-	if ( location === 'topic_page' && ! eventProperties.hasOwnProperty( 'tag' ) ) {
```

Comment:
Hmm, not sure why I added this [in the first place](https://github.com/Automattic/wp-calypso/commit/863e38908b87ed26dc910127c532a4ef65d67a3c). I can just add the `tag` property when calling `recordTrack()`, right?